scripts/migrate-all-data-to-cosmos.js

```diff
@@ -0,0 +1,235 @@
+    });
+
+    return recordsMissing;
+  } catch (err) {
+    logEmitter.emit("info", `findMissingRecords failed - ${err}`);
+  }
+};
+
+const insertCosmosRecord = async (fsaRn) => {
+  try {
+    const reg = await getRegistrationByFsaRn(fsaRn);
+    const {
+      fsa_rn,
+      collected,
+      collected_at,
+      createdAt,
+      direct_submission
+    } = reg.dataValues;
+
+    const establishment = await getEstablishmentByRegId(reg.dataValues.id);
+    const operator = await getOperatorByEstablishmentId(
+      establishment.dataValues.id
+    );
+    const partners = await getAllPartnersByOperatorId(operator.dataValues.id);
+    const premise = await getPremiseByEstablishmentId(
+      establishment.dataValues.id
+    );
+    const activities = await getActivitiesByEstablishmentId(
+      establishment.dataValues.id
+    );
+    const declaration = await getDeclarationByRegId(reg.dataValues.id);
+
+    const lcConfigCollection = await establishConnectionToCosmos(
+      "config",
+      "localAuthorities"
+    );
+    const hygieneAndStandards = await getLcContactConfig(
+      reg.dataValues.council
+    );
+    const { _id } = await findCouncilByUrl(
+      lcConfigCollection,
+      reg.dataValues.council
+    );
+
+    const notifications = [
+      {
+        time: createdAt,
+        sent: true,
+        type: "LC",
+        address: hygieneAndStandards.hygiene
+          ? hygieneAndStandards.hygiene.local_council_notify_emails
+          : hygieneAndStandards.hygieneAndStandards.local_council.notify_emails
+      },
+      hygieneAndStandards.standards
+        ? {
+            time: createdAt,
+            sent: true,
+            type: "LC",
+            address: hygieneAndStandards.standards.local_council_notify_emails
+          }
+        : [],
+      {
+        time: createdAt,
+        sent: true,
+        type: "FBO",
+        address: operator.dataValues.operator_email
+          ? operator.dataValues.operator_email
+          : operator.dataValues.contact_representative_email
+      },
+      declaration.dataValues.feedback1
+        ? [
+            {
+              time: createdAt,
+              sent: true,
+              type: "FBO_FB",
+              address: operator.dataValues.operator_email
+                ? operator.dataValues.operator_email
+                : operator.dataValues.contact_representative_email
+            },
+            {
+              time: createdAt,
+              sent: true,
+              type: "FD_FB",
+              address: (process.env.NODE_ENV = "prod"
+                ? "FutureDelivery@food.gov.uk"
+                : "fsatestemail.valid@gmail.com")
+            }
+          ]
+        : []
+    ];
+
+    const completeCacheRecord = Object.assign(
+      {},
+      {
+        "fsa-rn": fsa_rn,
+        collected: collected,
+        collected_at: collected_at,
+        reg_submission_date: createdAt,
+        directLcSubmission: direct_submission,
```

Comment:
Looks like the standard format is to use underscores, is now a good time to change directLcSubmission to match?

Reply:
Yeah that's a good point, okay will go with the underscore then